src/object/srv_obj.c

```diff
@@ -2340,7 +2340,8 @@ ds_obj_tgt_update_handler(crt_rpc_t *rpc)
 			  (orw->orw_bulks.ca_arrays != NULL ||
 			   orw->orw_bulks.ca_count != 0) ? true : false);
 	if (rc != 0)
-		D_CDEBUG(rc == -DER_INPROGRESS || rc == -DER_TX_RESTART, DB_IO, DLOG_ERR,
+		D_CDEBUG(rc == -DER_EXIST || rc == -DER_NONEXIST || rc == -DER_INPROGRESS ||
```

Comment:
Can be more accurate to distinguish conditional insert or update, that can be done in other PR, such as https://github.com/daos-stack/daos/pull/7598